policyengine_us/parameters/gov/irs/credits/estate/base.yaml

```diff
@@ -54,4 +54,6 @@ values:
   2022-01-01: 12_060_000  
   2023-01-01: 12_920_000  
   2024-01-01: 13_610_000
-  2025-01-01: 13_990_000  
+  2025-01-01: 13_990_000
+  # use 2017 value as base and cpi_w (2026Q1 projection) for uprating
+  2026-01-01: 7_460_000 # amount = 5490000 * 321.701 (2026Q1) / 236.854 (2017-01-01)
```

Comment:
I think it should be 5000000 * 321.701 (2026Q1) / 236.854 (2017-01-01)

Since law states to replace the 2017 value with 5,000,000